pages/apprenticeship.js

```diff
@@ -59,12 +59,13 @@ export default function Apprenticeship() {
 
                 <br />
 
-                <p>-Django</p>
-                <p>-React</p>
-                <p>-MySQL</p>
-                <p>-LESS CSS</p>
-                <p>-Git Workflow</p>
-
+                <ul>
```

Comment:
Can we change the list of skills to be also defined as a list similar to nav links, and render it using a for loop?

And provide URLs for each.

I would also like to see a 3-column grid (https://getbootstrap.com/docs/5.0/layout/grid/) with 3 lists. The heading for each list and the contents should be:

- Web and Frontend
  - JavaScript
  - CSS
  - HTML
  - NextJS <nextjs.org>
  - ReactJS <reactjs.org>
  - LESS CSS <http://lesscss.org/>
- Backend
  - Python
  - Django <https://www.djangoproject.com/>
  - Linux
  - BASH and other Unix utilities
- Server
  - MySQL <https://www.mysql.com/>
  - Redis <https://redis.io/>
  - RabbitMQ <https://www.rabbitmq.com/>